fedbiomed/researcher/experiment.py

```diff
@@ -1506,7 +1506,9 @@ def run_once(self, increase: bool = False, test_after: bool = False) -> int:
             )
             # FIXME: Access TorchModel through non-private getter once it is implemented
             aggregated_params: Dict[str, Union[torch.tensor, np.ndarray]] = (
-                self._job.training_plan._model.unflatten(flatten_params)
+                self._job.training_plan._model.unflatten(weights_vector=flatten_params,
+                                                         model_params=self.training_plan().after_training_params(
```

Comment:
Hi @sharkovsky, 
`after_training_params` is a method reserved to be used on the node side. In the beginning, it is developed to revert to keys of the model parameters after DP. However, it evolved in time :). I don't think that it is good idea to call on the researcher side since it executes some DP action that expects to be called on the node side.

Reply:
I agree, but actually there are some spots where we are using `training_plan().after_training_params`  in the Experiment

Reply:
I will look into this as soon as possible, I had a hunch that this might be a not ideal but I remember thinking that the alternatives were much more complicated... I'll come back to you when I have a clearer idea and can suggest a better way.